src/cmake/ie_parallel.cmake

```diff
@@ -34,7 +34,11 @@ endfunction()
 
 macro(ov_find_package_tbb)
     if(THREADING STREQUAL "TBB" OR THREADING STREQUAL "TBB_AUTO" AND NOT TBB_FOUND)
-        set(_ov_minimal_tbb_version 2017.0)
+        if (CONAN_EXPORTED)
+            set(_ov_minimal_tbb_version 2020.1)
+        else()
+            set(_ov_minimal_tbb_version 2017.0)
+        endif()
```

Comment:
why do we need to change minimal version?
Even TBB comes from Conan, we still can build with 2017.0 or higher

Reply:
Unfortunately, it looks that the `find_package` doesn't use `2017.0` as minimal version, but as exact version. I've used 2020.1 which is the same version which cmake download when doesn't find a package

Reply:
You are right. I've tried to check it again. I removed all Conan packages and installed them again. Now I see that with the flag `ENABLE_SYSTEM_TBB="ON"` everything works fine. I close the ticket